lib/vast/Conversion/Core/ToLLVM.cpp

```diff
@@ -104,7 +108,12 @@ namespace vast
 
                 cond_br_lhs(rewriter, op.getLoc(), cmp_lhs, rhs_block, end_block);
 
-                rewriter.setInsertionPointToEnd(rhs_block);
+                // In the case that `rhs` consists of multiple blocks we need to
+                // insert to the last block…
```

Comment:
...?

Reply:
I tried to update the comment. Is it now explained better or should I try more?